l10n_eu_nace/partner.py

```diff
@@ -19,22 +19,21 @@
 #
 ##############################################################################
 
-from openerp.osv import osv
+from openerp import models, api
 
 
-class PartnerCategory(osv.Model):
+class PartnerCategory(models.Model):
     """Let users search on code without a dot"""
     _inherit = 'res.partner.category'
 
-    def name_search(self, cr, uid, name, args=None, operator='ilike',
-                    context=None, limit=80):
+    @api.model
+    def name_search(self, name, args=None, operator='ilike', limit=100):
         """When no results are found, try again with an additional "."."""
-        results = super(PartnerCategory, self).name_search(
-            cr, uid, name, args=args, operator=operator, context=context,
-            limit=limit)
+        results = super(PartnerCategory, self).name_search(name, args=args,
```

Comment:
This is not PEP8 valid. Please put all arguments in the following line with one indentation more.